### ISSUE TITLE
Geoserver Security JDBC as role service option

### ISSUE THREAD

**Issue by ghelobytes** [geoserver/geoserver-cloud#562]
### Geoserver Security JDBC

The classic Geoserver has an option of creating a role service using JDBC. This option is not currently available on Geoserver cloud. Is there an architectural difference between the two (classic vs. cloud) that's why this isn't available by default?

Is it possible to add this option to Geoserver cloud?

So far I tried to add it as a dependency in `pom.xml` as indicated here https://github.com/camptocamp/geoserver-cloud-geoserver/packages/2022730 but that didn't really seem to work:

```
<dependency>
  <groupId>org.geoserver.security</groupId>
  <artifactId>gs-sec-jdbc</artifactId>
  <version>2.26.0.0</version>
</dependency>
```

| Geoserver classic | Geoserver cloud |
|----------|-----------------------|
|  <img src="https://github.com/user-attachments/assets/dfcf6534-3dae-4369-80b2-8beea81a32f6" width=300 /> | <img src="https://github.com/user-attachments/assets/e9539dec-d820-4e51-a933-fa9818cda08c" width=300 /> |
| <img src="https://github.com/user-attachments/assets/ef7802f4-481c-4582-bd26-732f1ffe5a22" width=300 /> | |

**Comment by groldan** [comment]
Hi @ghelobytes, indeed both the JDBC and LDAP security plugins aren't integrated. No reason other than having not needed them so far.

I just did, one this pull request #564 is merged you can try with the 1.9-SNAPSHOT docker images.

**Comment by ghelobytes** [comment]
> Hi @ghelobytes, indeed both the JDBC and LDAP security plugins aren't integrated. No reason other than having not needed them so far.
> 
> I just did, one this pull request #564 is merged you can try with the 1.9-SNAPSHOT docker images.

Thank you! I'll give this a try for sure. Is there a general guidelines/pattern on how to add plugins/extensions?
Or is it unique or a case to case basis?

**Comment by ghelobytes** [comment]
The PR https://github.com/geoserver/geoserver-cloud/pull/564 by @groldan has resolved this issue. Thank a lot!

Just a note that it will be even greater if we can figure out a guidelines/patterns/steps on how to port over plugins/extensions.
Closing this issue.
